src/aof.c

```diff
@@ -2471,7 +2471,11 @@ int rewriteAppendOnlyFileBackground(void) {
         char tmpfile[256];
 
         /* Child */
-        serverSetProcTitle("valkey-aof-rewrite");
+        if (server.server_symlink) {
```

Comment:
It's not necessary to store this bool in the server struct. We have `exec_argv` already.

```suggestion
        if (strstr(server.exec_argv[0], "redis-server") != NULL) {
```

Reply:
Thanks for sharing about exec_argv.